### ISSUE TITLE
Design fall-back certificate renewal

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/api#24]
**tl;dr:** we need a way for devices that are offline for an extended period and subsequently fails to renew its certificate using mTLS with some kind of one-time method.

This is a complicated task, as it needs to be bullet proof.

Here are some situations/edge cases that we need to think about:

 * What is the workflow for a device going offline and then comes back online 10 days after the device has expired?
 * What happens if the device performs a 'factory reset'?
 * What happens if a device is 'regifted' (i.e. changes ownership)?
 * What happens if somone clones the device?

Some potential solutions/technologies may include:

 * A long-time certificate that is only usable for key renewals
 * JSON Web Token
 * One-time tokens
